package.json

```diff
@@ -64,6 +64,7 @@
     "jest": "27.4.7",
     "jest-canvas-mock": "2.4.0",
     "postcss": "8.4.5",
+    "sass": "1.54.0",
```

Comment:
Do you need to use `sass`? Could we just use only postCSS?
If you just need to use nested, you can try [Tailwind Nesting](https://tailwindcss.com/docs/using-with-preprocessors#nesting) plugin

Reply:
I use `sass` with the CSS module, and it's simpler compared to setup PostCSS to work with the CSS module.